### DISCUSSION TITLE
Unlock not working Huyndai Tucson

### DISCUSSION THREAD

**Discussion by Zerobug01** [Hyundai-Kia-Connect/kia_uvo#809] [Q&A]
Hi,
I am new to HA and recentyl installed this integration.
I can see my cars infor in HA and i am able to lock my car from HA, however i unlocking does not seems to work or i am doing something wrong.
once the car is locked i again click on the lock icon to try and unlock the car, it seems that HA is sending a command to my car since i can see the 4 orange signal lights going on for a second but still my car is locked.

any help woudl be appreciated.

**Comment by cdnninja** [comment]
Please create an issue for this and follow the template to include the details we need to investigate.
